README.md

```diff
@@ -1,68 +1,137 @@
 
 1. Fork the repo to your own account.
-2. Clone: `git@github.com:your-username/take-my-picture.git`
-3. Navigate into the directory: `cd take-my-picture`
+2. Clone: `git@github.com:your-username/PiMotions.git`
+3. Navigate into the directory: `cd PiMotions`
+4. Rename the `.env.example` file to `.env`
 
-### 2. Get the app running locally
-1. `mv server/local_config.template.py server/local_config.py`
-2. `docker build -t <your_container_name> .`
-3. `docker run -d -p 80:80 <your_container_name>:latest`
-4. navigate to `localhost:80` in your browser, you should see the app running there
+### 2. Download some tools
 
-### 3. Set up a Resin.io account and application
+1. Download [Docker](https://www.docker.com/community-edition)
+2. Download [Etcher](https://etcher.io/)
 
-To make deploying code on our Pi easy, we're going to use a service called [Resin.io](https://resin.io/). Resin.io will make it easy for us to deploy our code on our Pis.
+### 3. Set up and configure Resin.io application
 
-[More about Resin.io](https://docs.resin.io/understanding/understanding-code-deployment/)
+To make deploying code on our Pi easy, we're going to use a service called [Resin.io](https://resin.io/).
+To learn more about Resin.io, [click here](https://docs.resin.io/understanding/understanding-code-deployment/).
 
-1. Sign up an account at [Resin.io](https://resin.io/).
+1. Sign up an account at [Resin.io](https://resin.io/). Ideally, use your GitHub account for authentication.
 
 2. You'll see a prompt to create a new application.
     1. Set Device Type to "Raspberry Pi (v1 or Zero)"
     1. Give your application a name!
     1. Click "Create New Application"
 
-### 4. Set up your Pi!
-TODO: flash pi with resin image
+3. Configure the resinOS image by toggling on "Development" under "Select Edition" and "Wifi + Ethernet" under "Network Connection".
+4. Enter the WiFi SSID and passphrase, and then click "Download".
+
+While the resinOS image is downloading, do the following:
+
+5. Navigate to the ["Preferences" page in the Resin dashboard](https://dashboard.resin.io/preferences/sshkeys) to add your SSH keys. Either select "Import from GitHub" (if you authenticated with GitHub), or enter your public SSH key manually.
+
+You can read more about finding your SSH key or generating a new one [here](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/).
+You can check to see if you have an existing SSH key by [following the steps here](https://help.github.com/articles/checking-for-existing-ssh-keys/).
+
+6. Set device configuration variable by clicking on "Fleet Configuration". Add the following two device config variables:
+- NAME: `RESIN_HOST_CONFIG_gpu_mem` VALUE: `128`
+- NAME: `RESIN_HOST_CONFIG_start_x` VALUE: `1`
+
+These variables make sure that the camera works on the Pi.
+
+7. Add Resin as a remote. To do so, copy the `git remote add` command in the top right of your Resin dashboard. Run this command in a new tab in your terminal.
```

Comment:
maybe add a step to make sure they are PiMotions directory when they are doing this here